### ISSUE TITLE
Stops after around 1000 scrobbles

### ISSUE THREAD

**Issue by nedludd** [benfoxall/lastfm-to-csv#25]
The process stops at page 6 out of 157 pages of my scrobbles.  At first I thought it was a rate limiting thing, but it does the same with my virgin LastFM API key.

Maybe their API changed recently?  Worked a couple of weeks ago.

Is there a way to debug what's going on from the client side?

**Comment by VeXHarbinger** [comment]
![image](https://user-images.githubusercontent.com/830407/113358249-71edbe80-9313-11eb-9447-72257aa49fc9.png)

Works for me so I'd say it's PEBCAK, or you have tracks with some odd characters.  You could debug it client side by running the code locally or getting a dev API key and using postman to make the req.   Or in your browser you could see if there are encoding errors in the console.

**Comment by VeXHarbinger** [comment]
Oh and I had 528 pages

**Comment by VeXHarbinger** [comment]
> ![image](https://user-images.githubusercontent.com/830407/113358249-71edbe80-9313-11eb-9447-72257aa49fc9.png)
> 
> Works for me so I'd say it's a track with some odd characters. You could debug it client side by running the code locally or getting a dev API key and using postman to make the req. Or in your browser you could see if there are encoding errors in the console.
> [VeXHarbinger-April-2021.csv.txt](https://github.com/benfoxall/lastfm-to-csv/files/6246669/VeXHarbinger-April-2021.csv.txt)

According to another report we are seeing the same error , but I can complete the task.
I downloaded with/in chrome if it helps

**Comment by nedludd** [comment]
Yeah, I managed to track down a track with some bad characters in the title.  

It wasn't showing up in the LastFM web list of my scrobbles.  I managed to find it with another LastFM downloader..

Maybe it's something in the `reqwest` library?  That's where it was hanging, `reqwest` got null input.
